honeybee_energy/ventcool/afn.py

```diff
@@ -0,0 +1,391 @@
+# coding=utf-8
+"""Functions to generate airflownetwork for a list of rooms."""
+from __future__ import division
+import math
+
+from .crack import AFNCrack
+from .opening import VentilationOpening
+from .crack_data import crack_data_dict
+
+
+# TODO: Move to ladybug_geometry.Base2DIn3D?
+def compute_bounding_box_x(geometries):
+    """Calculate minimum and maximum x coordinates of multiple geometries.
+
+    Note this function returns the coordinate extents relative to the standard
+    basis. If extents relative to a rotated bounding box is required, the geometries
+    need to be rotated to the standard basis before running this function.
+    """
+
+    geoms = geometries
+    min_x, max_x = geoms[0].min.x, geoms[0].max.x
+
+    for geom in geoms[1:]:
+        if geom.min.x < min_x:
+            min_x = geom.min.x
+        if geom.max.x > max_x:
+            max_x = geom.max.x
+
+    return min_x, max_x
+
+
+# TODO: Move to ladybug_geometry.Base2DIn3D?
+def compute_bounding_box_y(geometries):
+    """Calculate minimum and maximum y coordinates of multiple geometries.
+
+    Note this function returns the coordinate extents relative to the standard
+    basis. If extents relative to a rotated bounding box is required, the geometries
+    need to be rotated to the standard basis before running this function.
+    """
+
+    geoms = geometries
+    min_y, max_y = geoms[0].min.y, geoms[0].max.y
+
+    for geom in geoms[1:]:
+        if geom.min.y < min_y:
+            min_y = geom.min.y
+        if geom.max.y > max_y:
+            max_y = geom.max.y
+
+    return min_y, max_y
+
+
+# TODO: Move to ladybug_geometry.Base2DIn3D?
+def compute_bounding_box_z(geometries):
+    """Calculate minimum and maximum z coordinates of multiple geometries.
+
+    Note this function returns the coordinate extents relative to the standard
+    basis. If extents relative to a rotated bounding box is required, the geometries
+    need to be rotated to the standard basis before running this function.
+    """
+
+    geoms = geometries
+    min_z, max_z = geoms[0].min.z, geoms[0].max.z
+
+    for geom in geometries:
+        if geom.max.z > max_z:
+            max_z = geom.max.z
+        if geom.min.z < min_z:
+            min_z = geom.min.z
+
+    return min_z, max_z
+
+
+# TODO: Move to ladybug_geometry.Base2DIn3D?
+def compute_bounding_box_extents(geometries, axis_angle=0):
```

Comment:
The same goes for all of these. If you want to just leave them here for now, then please hide them from the public SDK.